Firstapp/app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
Should only have one root .gitignore.  Which means this should go in Firstapp/.gitignore.